scripts/AmnioteLH.json

```diff
@@ -0,0 +1,115 @@
+{
+    "citation": "Myhrvold, N.P., Baldridge, E., Chan, B., Sivam, D., Freeman, D.L. and Ernest, S.M., 2015. An amniote life-history database to perform comparative analyses with birds, mammals, and reptiles:Ecological Archives E096-269. Ecology, 96(11), pp.3109-000.",
+    "description": "Compilation of life history traits for birds, mammals, and reptiles.",
+    "name": "Amniote life History database",
+    "ref": "http://esapubs.org/archive/ecol/E096/269",
+    "shortname": "AmnioteDB",
+    "tables": {
+        "main": {
+            "cleanup": "Cleanup(correct_invalid_value, nulls=[-999])",
```

Comment:
I think that is correct, I will look at it more later and let you know if changes are needed